tools/run-fs.sh

```diff
@@ -130,7 +130,7 @@ case "${net}" in
         if [[ -z "${offline}" ]]
         then
             MC_PEER="insecure-mc://localhost:3200/,insecure-mc://localhost:3201/"
-            MC_TX_SOURCE_URL="http://localhost:4566/node-0-ledger/,http://localhost:4566/node-1-ledger/"
+            MC_TX_SOURCE_URL="file:///home/$USER/full-service/mobilecoin/target/docker/release/mc-local-network/node-ledger-distribution-0,file:///home/$USER/full-service/mobilecoin/target/docker/release/mc-local-network/node-ledger-distribution-1"
```

Comment:
This path is very specific to one's setup - is it possible to figure it out from the current directory this script is running in?

Reply:
Who is the main user of this script?

Reply:
What prevents something like `MC_TX_SOURCE_URL="file:///$PWD/mobilecoin/target/docker/r...` from working?